FruitFruit/FruitFruit/Controllers/HomeViewController.swift

```diff
@@ -6,12 +6,13 @@
 //
 
 import UIKit
+import FirebaseFirestore
 
 class HomeViewController: UIViewController {
     // MARK: - PROPERTIES
     var fruitOrders = [FruitOrder]()
     var fruitSaleInfos = [FruitSaleInfo]()
-    
+    let db = Firestore.firestore()
```

Comment:
```suggestion
    let datebase = Firestore.firestore()
```

많이 길지않고, 사람에 따라서 db의 의미를 모르는 사람이 있을 수 있습니다.
축약어를 지양하는게 어떨까요 ?